README.md

```diff
@@ -285,7 +289,9 @@ All the transitions in the contract can be categorized into three categories:
 
 All the callback transitions can be categorized into two categories:
 - _acceptance callback transitions_ will be called when this multi-signature contact is the `recipient` of the transaction.
+:warning: This contract is not ready to receive ZRC2 or other unrecognized tokens.
```

Comment:
This contract is ready to receive ZRC2 tokens because it has implemented the acceptance callback transitions, the warning should tell others that unless they have explicitly implemented the callbacks, they will not be able to receive it